public/data/CodingSpecies.json

```diff
@@ -1,12 +1,18 @@
 {
   "name": "CodingSpecies",
   "bio": "Making Apps using React.js, working on web and software enginering",
+  "bio": "Making Apps using React.js | Working on web and software engineering 👩‍💻 | Maintaining my organisation, App-Choreography 📛",
```

Comment:
```suggestion
  "bio": "Making Apps using React.js | Working on web and software engineering 👩‍💻 | Maintaining my organisation, App-Choreography 📛",
```

Reply:
You have a Duplicate object key -`" bio"`.Please remove the old one.
Otherwise LGTM.
Thanks for the Contribution :).

Reply:
Oh yes thank you @Pradumnasaraf